tools/ExtensionLibrary/DevHome.ExtensionLibrary/Views/ExtensionLibraryView.xaml

```diff
@@ -48,107 +48,111 @@
                                     Margin="0,0,0,28" />
 
                 <!-- Installed items -->
-                <Grid Margin="{StaticResource SettingsPageSubTitleMargin}">
-                    <TextBlock x:Uid="InstalledTextBlock"
+                <StackPanel x:Uid="InstalledItemsStackPanel" IsTabStop="True">

```

Comment:
I am not sure about the `IsTabStop` variable. It is one more button the user needs to press to navigate the UI. And it does not affect the narration of the internal controls.